lib/matplotlib/ticker.py

```diff
@@ -1296,7 +1296,7 @@ def __call__(self, x, pos=None):
         return self.fix_minus(s)
 
     def format_eng(self, num):
-        """
+        r"""
```

Comment:
I don't think this is right; the escape is in an output, so the interpreted string is correct.

Reply:
Thanks, you are right!